craft_providers/lxd/launcher.py

```diff
@@ -292,7 +292,7 @@ def launch(
     map_user_uid: bool = False,
     uid: Optional[int] = None,
     use_snapshots: Optional[bool] = None,
-    use_base_instance: Optional[bool] = False,
```

Comment:
I'm not sure why I made this optional (probably a copy/paste of the `use_snapshots` parameter).